### PR TITLE
`Development`: Add learning paths to feature list of the readme

### PR DESCRIPTION
**Pull request by MaximilianAnzinger**
#### General

- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

## Summary by CodeRabbit

- **New Features**
	- Introduced learning paths based on competency models to better align with student progress.
	- Integrated a new chatbot named Iris to assist with user queries.
	- Enhanced platform scalability to support more simultaneous users.
	- Improved customization options to cater to individual user preferences.

- **Focus Areas**
	- Continued emphasis on high user satisfaction.
	- Maintained open-source status for community contributions and transparency.

### CODE REVIEW

**Review by N0W0RK — DISMISSED** [review]
Code :+1:

File: README.md [hunk 1]
```diff
@@ -43,12 +43,13 @@ Artemis brings interactive learning to life with instant, individual feedback on
...
-20. **[Scalable](https://docs.artemis.cit.tum.de/admin/scaling/)**: Artemis scales to multiple courses with thousands of students simultaneously using it. In fact, the largest course had 2,400 students. Administrators can easily scale Artemis with additional build agents in the continuous integration environment.
-21. **[High user satisfaction](https://docs.artemis.cit.tum.de/user/user-experience/)**: Artemis is easy to use, provides guided tutorials. Developers focus on usability, user experience, and performance.
-22. **Customizable**: It supports multiple instructors, editors, and tutors per course and allows instructors to customize many course settings
-23. **[Open-source](https://docs.artemis.cit.tum.de/dev/open-source/)**: Free to use with a large community and many active maintainers.
+18. **[Learning paths](https://docs.artemis.cit.tum.de/user/adaptive-learning/adaptive-learning-student.html#learning-paths)**: Based on the competency model and students individual progress, Artemis creates learning paths that guide students through the course content.
```
Comment (JohannesStoehr) [inline README.md:46]:
```suggestion
18. **[Learning paths](https://docs.artemis.cit.tum.de/user/adaptive-learning/adaptive-learning-student.html#learning-paths)**: Based on the competency model and students' individual progress, Artemis creates learning paths that guide students through the course content.
```

**Review by JohannesStoehr — CHANGES_REQUESTED** [review]

**Review by N0W0RK — APPROVED** [review]

**Review by JohannesStoehr — APPROVED** [review]

**Review by JohannesStoehr — APPROVED** [review]

**Review by Jan-Thurner — APPROVED** [review]
lgtm

**Review by egekurt123 — APPROVED** [review]
LGTM

**Review by krusche — APPROVED** [review]

**Pull request merged**